jetstream/cli.py

```diff
@@ -560,8 +561,10 @@ def run(
         dataset_id=dataset_id,
         bucket=bucket,
         date=date,
-        experiment_slugs=[experiment_slug] if experiment_slug else All,
-        configuration_map={experiment_slug: config_file} if experiment_slug and config_file else {},
+        experiment_slugs=experiment_slug if experiment_slug else All,
+        configuration_map={experiment_slug[0]: config_file}
+        if experiment_slug and config_file
+        else {},
```

Comment:
Should we throw an exception if someone tries to pass more than one experiment slug with a custom config (or alternatively map the config to every experiment and log a warning to console)?

Reply:
yeah, I like the idea of throwing an exception